decomp/semantics/uds.py

```diff
@@ -54,18 +56,24 @@ class UDSCorpus(PredPattCorpus):
 
     def __init__(self,
                  graphs: Optional[PredPattCorpus] = None,
+                 docs: Dict[str, 'UDSDocument'] = None,
                  annotations: List['UDSAnnotation'] = [],
                  version: str = '1.0',
                  split: Optional[str] = None):
 
         self.version = version
 
-        self._corpus_paths = {splitext(basename(p))[0].split('-')[-1]: p
+        self._sentences_paths = {splitext(basename(p))[0].split('-')[-1]: p
                               for p
                               in glob(os.path.join(self.__class__.DATA_DIR,
                                                    version,
-                                                   '*.json'))}
+                                                   'uds-ewt-sentences*'))}
```

Comment:
This sort of glob worries me:  it is effectively munging structured filenames. Can we maybe separate the sentence- and document-level annotations into separate directories to explicitly represent that structure?

Reply:
To clarify: You're asking that both the actual annotations (currently in the `annotations/` directory) and the full JSON for the built graphs be split out into separate directories?

Reply:
I'm open to whatever structure you think makes sense. My suggestion would be to either:

1. Have two subdirectories of `data/1.0/annotations`: `data/1.0/annotations/sentence` and `data/1.0/annotations/document`, where the JSON in each directory contains only sentence- or document-level annotations, respectively. Then, the built sentence- and document-level graphs could just be dumped to `data/1.0` as I believe is currently happening.
2. Have two subdirectories of `data/1.0`: `data/1.0/sentence` and `data/1.0/document`, where each subdirectory has a subsubdirectory `annotations` where the corresponding annotations live. Then, the built sentence- and document-level graphs could be dumped to their corresponding subdirectory.

I don't really have a preference.